pkg/cmd/options/network.go

```diff
@@ -58,6 +60,9 @@ func (o *NetworkOptions) Complete(ctx context.Context, args []string) error {
 	if certKeyPath := os.Getenv(constants.ClientCertKeyEnvVar); certKeyPath != "" {
 		o.ClientCertKeyPath = certKeyPath
 	}
+	if o.Concurrency < 1 {
```

Comment:
Not really. We don't have a lot of the usual number-of-threads concurrency worries since there's at most one goroutine per layer and if it crashes you can just rerun it with a sane number.